ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.16-lp-rosa-classic.yaml

```diff
@@ -18,11 +18,11 @@ base_images:
   tests-private:
     name: tests-private
     namespace: ci
-    tag: "4.16"
+    tag: "4.15"
   tests-private-ui:
     name: tests-private-ui
     namespace: ci
-    tag: "4.16"
+    tag: "4.15"
```

Comment:
I guess we need to run test-private 4.15 before Logging 6.0 is released in OCP 4.16

Reply:
Okay, we will first test in this PR.